App/Files/FilesUtility/readAndWriteToFiles.js

```diff
@@ -1,13 +1,36 @@
+const fs = require("fs");
+
+const content = "some content!";
+
+const path = "./Output.txt";
+
+//lets see how to creat a function that will write the file
 function* writeToTextFile() {
-    fs.writeFile('Output.txt', data, (err) => {
+    fs.writeFile('./Output.txt', content, (err) => {
+        // if an error occurs 
+        if (err) {
+
+            console.log(err);
+
+        }
+        else {
+            // if no error
+            console.log("file written successfully");
```

Comment:
```suggestion
           return "file written successfully"
```

Reply:
we don't need to console log in production. You can use `return` statements instead. When testing on you local machine you are free to use `console.log` function